code/controllers/CommentingController.php

```diff
@@ -629,7 +629,7 @@ public function doPostComment($data, $form)
         } elseif (!$comment->Moderated) {
             // Display the "awaiting moderation" text
             $holder = $this->getOption('comments_holder_id');
-            $hash = "{$holder}_PostCommentForm_error";
+            $hash = "moderated";
```

Comment:
Why was this not included in https://github.com/silverstripe/silverstripe-comments/pull/177?